modules/apps/27-interchain-accounts/controller/types/msgs_test.go

```diff
@@ -17,6 +18,12 @@ import (
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
+type TypesTestSuite struct {
```

Comment:
you should be able to avoid adding test suite structs in all tests by using the following and registering the module's `AppModuleBasic`:

```golang
encodingCfg := moduletestutil.MakeTestEncodingConfig(ica.AppModuleBasic{})

// ...
signers, _, err := encodingCfg.Codec.GetSigners(msg)
```

Reply:
thank @damiannolan, I updated it